instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/utils.rb

```diff
@@ -53,7 +53,17 @@ def opname(operation, multi)
 
       def format_command(operation, args)
         placeholder = "#{operation} BLOB (OMITTED)"
-        command = [operation, *args].join(' ').strip
+
+        command = +operation.to_s
+        args.flatten.each do |arg|
+          str = arg.to_s
+          if str.bytesize >= CMD_MAX_LEN
```

Comment:
We have the `if` which checks if the `arg` bytesize exceeds limitation. In line 68, we are truncating the total concentrated command lines. I am thinking could we do something like:

```
if command.bytesize >= CMD_MAX_LEN
  break
elsif !str.empty?
  command << ' ' << str
end

command = OpenTelemetry::Common::Utilities.utf8_encode(command, binary: true, placeholder: placeholder)
```

We can remote line 68?

Reply:
Yeah, I should have made it clear that I made no attempt at optimizing the most out of it here, it's just a quick fix for a pathological case.